src/pymor/models/interface.py

```diff
@@ -254,13 +255,16 @@ def _compute_output_error_estimate(self, solution, mu=None, **kwargs):
         """
         if self.error_estimator is None:
             raise ValueError('Model has no error estimator')
-        return self.error_estimator.estimate_output_error(solution, mu, self, **kwargs)
+        return self.error_estimator.estimate_output_error(solution, mu, self,
+                                                          return_vector=return_vector,
+                                                          **kwargs)
 
     _compute_allowed_kwargs = frozenset()
 
     def compute(self, solution=False, output=False, solution_d_mu=False, output_d_mu=False,
                 solution_error_estimate=False, output_error_estimate=False,
-                output_d_mu_return_array=False, *, mu=None, input=None, **kwargs):
+                output_d_mu_return_array=False, output_error_estimate_return_vector=False,
+                *, mu=None, input=None, **kwargs):
```

Comment:
There is probably a doc entry missing for `output_error_estimate_return_vector`.